distros/README.md

```diff
@@ -0,0 +1,38 @@
+# Distribution specific Dockerfiles
+
+## Requirements
+
+### Dockerfile name
+The format should be `Dockerfile.{distribution name}`.  
+
+### Keep your images up-to-date
+Use the distribution package manager to keep your image up-to-date.
+
+### REPOSITORY
+Add a `REPOSITORY` comment with the URL to your GitHub repository where the Dockerfile is present.   
+`# REPOSITORY <GitHub repository>`  
+
+### MAINTAINER
+Add the `MAINTAINER` instruction and your contact details, GitHub aliases are acceptable.   
+
+## Example Dockerfile
```

Comment:
Minor sugegstion: How about just refer to the `Dockerfile.alpine` here like "For an example Dockerfile, please refer to Dockerfile.alpine". This can avoid the repetitive text in the Readme